src/java/voldemort/server/VoldemortConfig.java

```diff
@@ -406,6 +407,7 @@ public VoldemortConfig(Props props) {
 
         this.modifiedProtocol = props.getString("readonly.modify.protocol", null);
         this.modifiedPort = props.getInt("readonly.modify.port", -1);
+        this.bouncyCastleEnabled = props.getBoolean("readonly.bouncy.castle.enabled", false);
```

Comment:
I thought about the option you said. If we'd like to use BouncyCastleProvider for all SSL encryption/decryption, yes, we should initialize here. Will fix it.